plugins/html-elements-plugin/src/main/java/ru/sbtqa/tag/pagefactory/utils/HtmlReflectionUtils.java

```diff
@@ -382,23 +380,24 @@ public static void executeMethodByTitleInBlock(Page page, String blockPath, Stri
                 }
             }
         }
- 
-        isUsedBlock = true; 
-        usedBlock = block; 
-        List<Method> methodList = getDeclaredMethods(page.getClass()); 
-        for (Method method : methodList) { 
-            if (isRequiredAction(method, actionTitle)) { 
-                try { 
-                    method.setAccessible(true); 
-                    MethodUtils.invokeMethod(page, method.getName(), parameters); 
-                    return; 
-                } catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e) { 
-                    throw new FactoryRuntimeException(String.format("Failed to execute method '%s' in the following block: '%s'", 
-                            actionTitle, blockPath), e); 
-                } 
-            } 
+
+        // TODO WTF??
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)